core/trino-main/src/main/java/io/trino/cost/FilterStatsCalculator.java

```diff
@@ -51,6 +51,7 @@
 import javax.annotation.Nullable;
```

Comment:
Add description in commit message what does it fix

Reply:
Added the description in commit. Thanks for pointing it out.